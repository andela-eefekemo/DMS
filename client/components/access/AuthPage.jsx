import React, { Component } from 'react';
import Header from '../include/Header';

/**
 * @class LandingContainer
 * @extends {Component}
 */
class AuthPage extends Component {
  constructor(props) {
    super(props);
  }
  /**
   * @returns {jsx} -
   * @memberof LandingContainer
   */
  render() {
    return (
      <div>
        <Header match={this.props.match} />
        <div className="container authpage">
          <div className="row auth-card">
            {this.props.children}
          </div>
        </div>
      </div>
    );
  }
}

export default AuthPage;
