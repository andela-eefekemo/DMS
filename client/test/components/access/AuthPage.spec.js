/* global expect jest test */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AuthPage from '../../../components/access/AuthPage';

jest.dontMock('../../../components/access/AuthPage');

describe('AuthPage Component', () => {
  test('should match the AuthPage snapshot', () => {
    const match = {
      path: ''
    };
    const component = shallow(<AuthPage match={match} />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
