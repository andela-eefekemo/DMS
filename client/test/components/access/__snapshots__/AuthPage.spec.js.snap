// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthPage Component should match the AuthPage snapshot 1`] = `
<div>
  <Connect(withRouter(Header))
    match={
      Object {
        "path": "",
      }
    }
  />
  <div
    className="container authpage"
  >
    <div
      className="row auth-card"
    >
      <Switch>
        <Route
          component={[Function]}
          exact={true}
          path="/signin"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/signup"
        />
      </Switch>
    </div>
  </div>
</div>
`;
